pkg/config/config.go

```diff
@@ -155,14 +155,14 @@ func readConfigOrSecret(secretName, configPath string, v *viper.Viper) error {
 		}
 	}
 
-	for k, v := range v.AllSettings() {
```

Comment:
I think you replaced the wrong `v`

Reply:
https://github.com/IBM/argocd-vault-plugin/pull/122/commits/143b13dc9fff6580239b4701140452a8d2dfb681#diff-a3d824da3c42420cd5cbb0a4a2c0e7b5bfddd819652788a0596d195dc6e31fa5R158

`viperValue` is the value for the given Viper key, seems right to me?

Reply:
I guess i got confused. Since `v` is the viper instance. And then `viperValue` is the viper settings. I guess it's not a big deal.